test/server/routes/__mocks__/index.js

```diff
@@ -1,9 +1,11 @@
 import * as authorData from './authorData';
 import * as userData from './userData';
 import * as bookData from './bookData';
+import * as borrowData from './borrowData';
 
 export default {
   authorData,
   userData,
-  bookData
+  bookData,
+  borrowData
```

Comment:
No need for a borrowData file. Just put the objects in the bookData file

Reply:
Noted, Done